models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -387,10 +387,24 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     ## plantWoodInit gC/m2
     plant_wood_vars <- c("AbvGrndWood", "abvGrndWoodFrac", "coarseRootFrac", "fineRootFrac")
     if (all(plant_wood_vars %in% ic.names)) {
-  
-      
       # reconstruct total wood C
       wood_total_C <- IC$AbvGrndWood / IC$abvGrndWoodFrac
+      #Sanity check
+      if (is.na(wood_total_C) | is.infinite(wood_total_C) | is.nan(wood_total_C) | wood_total_C <0) {
```

Comment:
```suggestion
      if (is.na(wood_total_C) | is.infinite(wood_total_C) | is.nan(wood_total_C) | wood_total_C < 0) {
```

Reply:
Why the mix of inf, NA and NaN?

Reply:
@para2x:  maybe switch to !(is.finite(wood_total_C))???? That would capture anything that isn't a real number.